<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdTableDialog</class>
 <widget class="QDialog" name="PdTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>보정 테이블</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="cbChannel"/>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>44</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Reference</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="editRef"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>(dBm)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>44</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="editVolt">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>(mV x 10)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="btnSetVolt">
     <property name="text">
      <string>Set Voltage</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QPushButton" name="btnWriteTable">
     <property name="text">
      <string>Write Table</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="10">
    <widget class="QTableWidget" name="tableWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>65</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Voltage</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>0 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-5 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-10 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-15 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-20 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-25 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-30 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-35 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-40 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-45 dB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>-50 dB</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
